db/wpdb.ts

```diff
@@ -549,21 +545,9 @@ export const getBlockContent = async (
         }
       }
     `
+    const post = await graphqlQuery(query, { id })
 
-    const response = await fetch(WP_GRAPHQL_ENDPOINT, {
-        method: "POST",
-        headers: {
-            "Content-Type": "application/json",
-            Accept: "application/json",
-        },
-        body: JSON.stringify({
-            query,
-            variables: { id },
-        }),
-    })
-    const json = await response.json()
-
-    return json.data.post?.content ?? undefined
+    return post.data.post?.content ?? undefined
```

Comment:
nit: would this ever be null? if not then maybe dont need the right half?

Reply:
In the case of an empty post content, it can actually:
<img width="317" alt="Screenshot 2021-01-15 at 09 37 35" src="https://user-images.githubusercontent.com/13406362/104702967-0b470480-5717-11eb-85fb-c2677c8b772f.png">